core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -270,63 +273,71 @@ fun LPGenericElevatedCard(
  */
 @Composable
 fun LPGenericElevatedCardImage(
-    imageUrl: String,
+    imageUrl: String?,
     title: String,
     description: String,
     onClick: () -> Unit
 ) {
+    val dp2 = LeasePertTheme.sizes.middleSize
+    val dp8 = LeasePertTheme.sizes.smallerSize
+    val dp12 = LeasePertTheme.sizes.midSmallSize
+    val dp175 = LeasePertTheme.sizes.extraSize175
     Card(
-        shape = MaterialTheme.shapes.medium.copy(all = CornerSize(12.dp)),
+        border = BorderStroke(width = dp2, color = Drab),
```

Comment:
Instead of import the color Drab, import please the material design theme like this: `MaterialTheme.colorScheme.onPrimary` and so on for the rest of colors....